www/pages/blog/release/v0-28-0.md

```diff
@@ -0,0 +1,136 @@
+---
+label: 'blog'
+title: v0.28.0 Release
+template: blog
+---
+
+# Greenwood v0.28.0
+
+**Published: April 8, 2023**
+
+## What's New
+
+<img src="/assets/nodejs.png" style="display: block; width: 30%; margin: 0 auto;"/>
+
+It's release time again and the **Greenwood** team is excited to share some of the exciting new features available in this version, as we continue to make the experience of writing for the _**Full-Stack Web**_ easier for everyone! 🙌
+
+### Node 18
+
+With this release, Greenwood has set the minium version of NodeJS to 18.  This NodeJS release is especially profound to us as it brings with it (amongst other things) native support for the [**Fetch API**](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API).  You can now use `fetch` directly in your server side code like our SSR pages! 💯
+
+```js
+// src/pages/artists.js
+import '../components/card.js';
+
+export default class ArtistsPage extends HTMLElement {
+  async connectedCallback() {
+    const artists = await fetch('https://.../api/artists').then(resp => resp.json());
+    const html = artists.map((artist) => {
+      const { name, imageUrl } = artist;
+  
+      return `
+        <wc-card>
+          <h2 slot="title">${name}</h2>
+          <img slot="image" src="${imageUrl}" alt="Picture of ${name}"/>
+        </wc-card>
+      `;
+    }).join('');
+
+    this.innerHTML = html;
+  }
+}
+```
+
+> _As a bonus worth mentioning, [JSON imports](https://simonplend.com/import-json-in-es-modules/) are now natively available using import assertions._
+
+
+### Embracing Web APIs
+
+What's great about the addition of the Fetch API is that it brings along with it many Web API friends like [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL), [`Request`](https://developer.mozilla.org/en-US/docs/Web/API/Request), and [`Response`](https://developer.mozilla.org/en-US/docs/Web/API/Response) just to name a few.  Greenwood has fully embraced this movement to adopting Web APIs not only throughout its code base, but basing entire user facing APIs around these standards as well.  Why invent an API when we get everything we need from the web, in Node, and all documented by MDN!
+
+This was especially beneficial to our [Resource Plugin API](/plugins/resource/) as it was already modeling this request / response behavior anyway, and so it was a natural fit to adopt these APIs. To give an idea of this transformation, here is a before snippet of Greenwood's internal plugin for handling CSS.
+```js
+// eslint-disable-next-line no-unused-vars
```

Comment:
eslint ignore rules should go outside the post as HTML comments, otherwise they show up in the code snippets